### ISSUE TITLE
Accessibility label missing on button

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1639]
The accessibility label is missing on "Add your first credential" as shown in the attached image.

![IMG_1479](https://github.com/bcgov/bc-wallet-mobile/assets/390891/a5b5dcab-24c3-4309-8fa3-4087402d09d4)
